build.gradle

```diff
@@ -79,10 +79,17 @@ def getModDir() {
     return project.getProperty('game.dir') + "/mods/" + modInfo().id
 }
 
-task copyToMods(type: Copy) {
+def isModPathDefined() {
     if (!project.hasProperty('game.dir')) {
         println "WARNING: Missing game.dir in gradle.properties. See gradle.properties.dist as an example."
         println "WARNING: Task execution stopped."
+        return false;
+    }
+    return true;
+}
+
+task copyToMods(type: Copy) {
+    if (!isModPathDefined()) {
```

Comment:
I am sure there is a better way of doing it, like conditional evaluaton of tasks.